### ISSUE TITLE
Add links to Pypi

### ISSUE THREAD

**Issue by odscjames** [gazpachoking/jsonref#44]
Can I suggest adding links from the Pypi page to the docs and Github?

If it was in setup.py you can do:

    project_urls={
        "Documentation": "https://aaaaa.readthedocs.io/en/latest/",
        "Issues": "https://github.com/aaaa/bbbb/issues",
        "Source": "https://github.com/aaa/bbbbb",
    },
    
 But I'm not sure how you do it in pyproject.toml

**Comment by gazpachoking** [comment]
Good call. I added it in 670d318c9050e0d700cd06ccd55d11d124cb767b. pypi is in maintenance mode right now, will push a post release when they are back to normal.

**Comment by odscjames** [comment]
Thank you!

**Comment by gazpachoking** [comment]
Pypi is back to normal, links are up!
